addon/templates/project/form.hbs

```diff
@@ -7,14 +7,14 @@
       <UkSpinner @ratio="3" />
     </div>
   {{else}}
-    <h2 class="uk-flex uk-flex-between">
```

Comment:
This already worked with alignment in the dummy app. Does kt-be have special button / h2 layout styling?

Reply:
I also did a quick test in sz and seems to be ok like this.

Reply:
No, this should be fine in BE as well.